git_machete/cmd.py

```diff
@@ -196,145 +57,19 @@ def pick(choices: List[str], name: str, apply_fmt: bool = True) -> str:
     return choices[index]
 
 
-def debug(cli_ctxt: CommandLineContext, hdr: str, msg: str) -> None:
-    if cli_ctxt.opt_debug:
-        sys.stderr.write(f"{bold(hdr)}: {dim(msg)}\n")
-
-
 # To avoid displaying the same warning multiple times during a single run.
 displayed_warnings: Set[str] = set()
 
 
 def warn(msg: str, apply_fmt: bool = True) -> None:
```

Comment:
Hmmm intuitively I'd say that `debug` and `warn` should be in the very same place (`utils`)

Reply:
Coz they are conceptually just different levels of logging